addons/hr_contract/views/hr_employee_views.xml

```diff
@@ -10,4 +10,28 @@
             </xpath>
         </field>
     </record>
+
+    <record id="view_hr_employee_graph_inherit" model="ir.ui.view">
+        <field name="name">hr.employee.graph</field>
```

Comment:
for inherited view  
`<record id="hr_employee_view_graph_inherit_hr_contract" model="ir.ui.view"> 
`
and we match name as well


https://www.odoo.com/documentation/16.0/contributing/development/coding_guidelines.html?highlight=naming#inheriting-xml